cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -90,7 +90,7 @@ public Outlet retrieveOutlet(String officeId, String locationId) {
             LOCATION_REF_T locRef = LocationUtil.getLocationRef(locationId, officeId);
             Configuration config = DSL.using(conn).configuration();
             PROJECT_STRUCTURE_OBJ_T outletStruct = CWMS_OUTLET_PACKAGE.call_RETRIEVE_OUTLET(config, locRef);
-            
+            // TODO: This may be causing tests to fail with pool size = 1.
```

Comment:
Should be fixed now with PR: https://github.com/USACE/cwms-data-api/pull/970

Reply:
Yep tests passed.